src/elements/common/sub-header/GridViewButton.scss

```diff
@@ -0,0 +1,8 @@
+.bdl-GridViewButton {
+    margin-left: 7px;
```

Comment:
Does this margin need to be baked into the Button component? Seems like it would be a style more suited for the consumer of this button